language/fr/fleet.lang

```diff
@@ -1,5 +1,275 @@
+$_Lang['fl_no_shortcuts']                        = 'Aucune cible personnalisée';
+$_Lang['fl_no_planets']                          = 'Pas de colonies';
+$_Lang['fl_your_planets']                        = 'Vos colonies';
+$_Lang['fl_your_shortcuts']                      = 'Vos cibles personnalisées';
+$_Lang['fl_add_shortcut']                        = 'Gestion des cibles personnalisées';
+$_Lang['moon_sign']                              = 'L';
+$_Lang['debris_sign']                            = 'CD';
+$_Lang['planet_sign']                            = 'P';
+
+$_Lang['DropdownList_Moon_sign']                 = 'L';
+$_Lang['fl_dropdown_select']                     = 'Sélectionner';
+
+// fleet2.php
+$_Lang['fl_ressources']                          = 'Ressources';
+$_Lang['fl_allressources']                       = 'Toutes les ressources';
+$_Lang['fl2_allResZero']                         = 'Désélectionner les ressources';
+$_Lang['fl_bad_fleet_speed']                     = 'Vitesse de vol invalide!';
+$_Lang['fl2_targeterror']                        = 'Coordonnées de destination invalides!';
+$_Lang['fl2_cantsendsamecoords']                 = 'La destination du vol ne peut pas être la même que le point d\'origine!';
+$_Lang['fl2_NoEnoughFuel']                       = 'Vous n\'avez pas assez de deutérium pour envoyer cette flotte!';
+$_Lang['fl2_Have2UseQuantumGate']                = 'Pour envoyer cette flotte, vous devez utiliser Quantum Gate, <br/> car vous n’avez pas assez de deutérium ou de capacité de charge suffisante sur vos navires!';
+$_Lang['fl2_NoEnoughStorage']                    = 'Vous n\'avez pas une capacité de charge suffisante pour le carburant nécessaire pour envoyer cette flotte!';
+$_Lang['fl2_FlyTimeInfo']                        = 'En cas d\'utilisation de Quantum Gate, le deuxième compteur indique le temps qu\'il faudra à cette flotte pour revenir au point d\'origine.';
+
+$_Lang['fl2_title']                              = 'Envoyer la flotte de  ';
+$_Lang['fl2_sendfromplanet']                     = 'une planète';
+$_Lang['fl2_sendfrommoon']                       = 'une lune';
+$_Lang['fl2_emptyplanet']                        = 'Inhabitée';
+$_Lang['fl2_target_abandoned_1']                 = 'Planète abandonnée';
+$_Lang['fl2_target_abandoned_3']                 = 'Lune abandonnée';
+$_Lang['fl2_debrisfield']                        = 'Champ de débris';
+
+$_Lang['fl2_flightTime']                         = 'Durée du vol <br/> (aller simple)';
+$_Lang['fl2_destination']                        = 'Destination';
+$_Lang['fl2_targetowner']                        = 'Propriétaire de la cible';
+
+$_Lang['fl2_SendButton']                         = 'Envoyer la flotte';
+
+$_Lang['fl2_confirm_allypact_attack']            = 'Voulez-vous vraiment attaquer un joueur qui fait partie d\'une alliance qui a un pacte avec l\'alliance dans laquelle vous êtes actuellement?';
```

Comment:
This string is passed to JavaScript code, therefore you have to escape the single quote sign with tripple slashes, not just one, like this:
```
'Voulez-vous vraiment attaquer un joueur qui fait partie d\\\'une alliance qui a un pacte avec l\\\'alliance dans laquelle vous êtes actuellement?';
```